src/LodeRunner/src/Services/LodeRunnerService.cs

```diff
@@ -235,8 +275,10 @@ private async Task<int> Start()
 
             if (this.config.RunLoop)
             {
+                var app = new App();
```

Comment:
I am not sure I understand why we need to create a new App ?

Reply:
If we do not create a new App here, the LodeRunner instance running command mode will shut down the LodeRunner instance running in client mode when it finishes the test.  This would prevent the instance running in client mode from staying alive and continuing to execute new test runs. This change came along with changing the BuildWebHost in Program.cs (in LodeRunner/src) to no longer be static and also have the cancellation token be passed as a parameter so it is not using the same token as the client mode instance.  Happy to discuss more.

Reply:
Thanks for the offline talk! As discussed, I moved the BuildWebHost function into the LodeRunnerService class since that is the only place where it is used (for RunLoop)